arangod/Transaction/Methods.cpp

```diff
@@ -1173,6 +1173,9 @@ void transaction::Methods::invokeOnAllElements(std::string const& collectionName
 
   TRI_voc_cid_t cid = addCollectionAtRuntime(collectionName);
   TransactionCollection* trxCol = trxCollection(cid, AccessMode::Type::READ);
+  if (trxCol == nullptr) {
```

Comment:
```suggestion
TRI_ASSERT(trxCol != nullptr)
```

Reply:
I think, even if due to a programming error, more precise error codes & messages will help here, at least with debugging.

Reply:
and waste cycles